exchanges/gateio/gateio.go

```diff
@@ -173,8 +173,8 @@ var (
 
 // Gateio is the overarching type across this package
 type Gateio struct {
+	Counter common.Counter // Must be first	due to alignment requirements
```

Comment:
This realignment due to the requirement for assignment alignment brings no excitement, only chastisement that my review indictment brought defilement 😭

Reply:
💯 🥇 😗